vars/unitTestPost.groovy

```diff
@@ -45,45 +52,50 @@ def call(Map config = [:]) {
     return
   }
 
-  junit testResults: config.get('testResults', 'test_results/*.xml')
+  def test_results = config.get('testResults', 'test_results/*.xml')
+  if (test_results != "") {
```

Comment:
When would this be empty?  If it never should, we shouldn't paper over it being empty by not running the `junit` step and preventing it from register the missing results.

Reply:
you're right, I'll fix it